crates/xilem_html/src/view.rs

```diff
@@ -155,38 +155,36 @@ where
         state: &mut Self::State,
         element: &mut Self::Element,
     ) -> ChangeFlags {
-        let mut change_flags = ChangeFlags::empty();
         match (prev, self) {
             (Either::Left(_), Either::Right(view)) => {
                 let (new_id, new_state, new_element) = view.build(cx);
                 *id = new_id;
                 *state = Either::Right(new_state);
                 *element = Either::Right(new_element);
-                change_flags |= ChangeFlags::STRUCTURE;
```

Comment:
I got into the habit of 'or-ing' change flags to make sure I always added rather than replaced flags in cases where there were multiple changes, but here it's fine to just return the changeflag directloy since it's equivalent.